Packs/NutanixHypervisor/Integrations/NutanixHypervisor/NutanixHypervisor.py

```diff
@@ -0,0 +1,1017 @@
+
+    In case response was successful, response will be a dict {'task_uuid': int}.
+
+    Args:
+        client (Client): Client object to perform request.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    vm_uuid = args.get('vm_uuid')
+    host_uuid = args.get('host_uuid')
+    transition = args.get('transition')
+
+    raw_response = client.nutanix_hypervisor_vm_power_status_change(vm_uuid, host_uuid, transition)
+
+    return CommandResults(
+        outputs_prefix='NutanixHypervisor.VMPowerStatus',
+        outputs_key_field='task_uuid',
+        outputs=raw_response,
+        raw_response=raw_response
+    )
+
+
+def nutanix_hypervisor_task_results_get_command(client: Client, args: Dict):
+    """
+    Poll tasks given by task_ids to check if they are ready.
+    Returns all the tasks from 'task_ids' list that are ready at the moment
+    Nutanix service was polled.
+    In case no task is ready, waits 'TIMEOUT_INTERVAL' (1 second) seconds and in case no task had finished,
+    returns a time out response.
+
+    In case response was successful, response will be a dict containing key 'completed_tasks_info'
+    which holds details about every completed task returned by Nutanix service.
+    In case response timed out, response will be a dict {'timed_out': True}.
+
+    Args:
+        client (Client): Client object to perform request.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    task_ids_list: List[str] = argToList(args.get('task_ids'))
+
+    raw_response = client.nutanix_hypervisor_task_results(task_ids_list)
+    outputs = copy.deepcopy(raw_response.get('completed_tasks_info', []))
+
+    readable_task_details_output: List[Dict] = []
+    for output in outputs:
+        task_id = output.get('uuid')
+        progress_status = output.get('progress_status')
+        readable_task_details_output.append({'Task ID': task_id, 'Progress Status': progress_status})
+        task_ids_list.remove(task_id)
+
+    for uncompleted_task_id in task_ids_list:
+        progress_status = 'In Progress' if task_exists(client, uncompleted_task_id) else 'Task Was Not Found'
+        readable_task_details_output.append({'Task ID': uncompleted_task_id, 'Progress Status': progress_status})
+        outputs.append({'uuid': uncompleted_task_id, 'progress_status': progress_status})
+
+    readable_output = tableToMarkdown('Nutanix Hypervisor Tasks Status', readable_task_details_output,
+                                      headers=['Task ID', 'Progress Status'])
+
+    add_iso_entries_to_dict(outputs)
+
+    final_outputs = [remove_empty_elements(output) for output in outputs]
+
+    return CommandResults(
+        outputs_prefix='NutanixHypervisor.Task',
+        outputs_key_field='uuid',
+        readable_output=readable_output,
+        outputs=final_outputs,
+        raw_response=raw_response
+    )
```

Comment:
Didn't we say we want to have the data of the task in the human-readable too? And then you don't need the readable_task_details_output veriable.